systemtest/conftest.py

```diff
@@ -36,7 +39,7 @@ def nginx(config):
 
 @pytest.fixture
 def postcheck(config):
-    """Return whether or not we can verify the file was saved (need access to S3)"""
+    """Return whether or not we can verify the file was saved (need access to S3/GCS)"""
```

Comment:
If we generalize this, then we don't have to update it in the future.

```suggestion
    """Return whether or not we can verify the file was saved (need access to storage)"""
```